frontend/www/js/omegaup/components/arena/Course.vue

```diff
@@ -11,7 +11,14 @@
       }}</sup>
     </template>
     <template #clock>
-      <div class="clock">{{ clock }}</div>
+      <div v-if="isAssignmentFinished" class="clock">{{ clock }}</div>
+      <div v-else-if="!deadline" class="clock">{{ INF }}</div>
+      <omegaup-countdown
+        v-else
+        class="clock"
+        :target-time="deadline"
+        @finish="now = Date.now()"
```

Comment:
```suggestion
        @finish="now = new Date()"
```

aquí hay que concordar los tipos con L224: `now` es un `Date`, no un `number`.

Reply:
Listo. También lo arreglé en el componente de Concursos